tests/test_time_series_graph.py

```diff
@@ -581,6 +581,44 @@ def test_from_adjacency_matrices(self):
                 self.assertLess(source.time_lag, destination.time_lag)
                 self.assertEqual(edge.get_edge_type(), EdgeType.DIRECTED_EDGE)
 
+        # test for the bugfix
```

Comment:
Can you explain a bit more what the bug was. Once this is merged no one will know what bugfix is being referred to. Why do these edge pairs act as a regression case.